docs/sources/helm-charts/mimir-distributed/release-notes/v5.1.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Grafana Mimir Helm chart version 5.1 release notes"
+menuTitle: "V5.1 release notes"
+description: "Release notes for Grafana Mimir Helm chart version 5.1"
+weight: 300
+---
+
+# Grafana Mimir Helm chart version 5.1 release notes
+
+Grafana Labs is excited to announce version 5.1 of the Grafana Mimir Helm chart, which is compatible with Grafana Mimir v2.10 or Grafana Enterprise Metrics (GEM) v2.10. The `mimir-distributed` Helm chart is the best way to install Mimir on Kubernetes.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For a comprehensive list of changes, see the [Helm chart changelog](https://github.com/grafana/mimir/tree/main/operations/helm/charts/mimir-distributed/CHANGELOG.md).
+
+## Important changes
+
+- Deployments now set `maxUnavailable` to 0 and `maxSurge` to a sensible value by default to avoid becoming unavailable during rollout. Please note that if you have
```

Comment:
```suggestion
- Deployments now set `maxUnavailable` to 0 and `maxSurge` to `15%` by default to avoid becoming unavailable during rollout. Please note that if you have
```